### DISCUSSION TITLE
how to chage BASE_URL?

### DISCUSSION THREAD

**Discussion by supwang** [ChatGPTNextWeb/NextChat#4092] [Q&A]
Hi, All:

Does anyone know how to change BASE_URL?
Thank you.

**Comment by H0llyW00dzZ** [comment]
if you are using `vercel`, it's in `environment-variables`

**Reply by supwang** [reply]
Thank you.
However, I did not find a place to change the BASE_URL in environment-variables.
![Screenshot 2024-02-21 at 11 18 54](https://github.com/ChatGPTNextWeb/ChatGPT-Next-Web/assets/4122964/7615944c-69d7-42d8-88e7-431b29028373)

**Reply by H0llyW00dzZ** [reply]
add one, then redeploy it

**Reply by supwang** [reply]
Thank you.
